### PR TITLE
include e-tag in restore requests

### PR DESCRIPTION
**Pull request by welansari**
Issue: bugfix/BB-581

### CODE REVIEW

File: .github/scripts/run_ft_tests.bash [hunk 1]
```diff
@@ -20,5 +20,4 @@ killandsleep () {
 }
 
 cd ${NODE_PATH}/@zenko/cloudserver && yarn run mem_backend & bash tests/utils/wait_for_local_port.bash $PORT 40
-./node_modules/.bin/nyc --clean --silent yarn run $1
-./node_modules/.bin/nyc report --report-dir "./coverage/$1" --reporter=lcov
+./node_modules/.bin/c8 --report-dir "./coverage/$1" --reporter lcov --clean yarn run $1
```
Comment (francoisferrand) [inline .github/scripts/run_ft_tests.bash]: why did you change this?

Comment (welansari) [reply .github/scripts/run_ft_tests.bash]: Just testing things out to see if maybe there was an issue in merging the coverage of c8 and nyc

File: .github/scripts/run_server_tests.bash [hunk 2]
```diff
@@ -16,3 +14,5 @@ killandsleep () {
 ./node_modules/.bin/c8 --report-dir "./coverage/$1" --reporter lcov yarn start &
 bash tests/utils/wait_for_local_port.bash $PORT 40
 yarn run $1
+killandsleep
+./node_modules/.bin/c8 report --report-dir "./coverage/$1" --reporter lcov
```
Comment (francoisferrand) [inline .github/scripts/run_server_tests.bash]: missing new line at EOF

**Review by francoisferrand — APPROVED** [review]

**Review by KillianG — APPROVED** [review]
